inc/saivlan.h

```diff
@@ -98,6 +98,12 @@ typedef enum _sai_vlan_attr_t
       * Value Range SAI_SWITCH_ATTR_VLAN_USER_META_DATA_RANGE */
     SAI_VLAN_ATTR_META_DATA,
 
+    /**
+      * The Multicast 802.1BR E-Channel Id (ECID), to be used in the flooding traffic
+      * sent by the CB to Port Extenders (PE). [sai_uint32_t] (CREATE_AND_SET) (default to 0)
+      */
+    SAI_VLAN_ATTR_FLOODING_ECID,
```

Comment:
@rsivasankar ECID is not a global concept, it is allocated within the scope of an Internal External Bridge. There could be multiple Internal External Bridges within a CB. So ECID can not be used globally. And I don't understand the purpose of this FLOODING_ECID. If it's for PE replication (multicast ECID), then it's wrong. A packet could be flooded to normal access ports and multiple multicast E-Channels at the same time. For CB replication, this attribute is not needed.

PS; Suggest adding new switch level attribute to indicate that the switch is a CB or PE.